tools/src/main/scala/scala/scalanative/interflow/MergeProcessor.scala

```diff
@@ -473,6 +481,8 @@ final class MergeProcessor(
 object MergeProcessor {
   case object Restart extends Exception with scala.util.control.NoStackTrace
 
+  private val MergeBlockOffset = 1000000L
```

Comment:
Do you think it might be a good idea to leave some hints here or at the top about the 1m limit?

Reply:
There is an assertion at the begging of the MergeProcessor constructor, but I can add some comments here as well. 
I don't think we shall ever reach such a big number again in any sane code, but better be safe than sorry